### PR TITLE
[Enhancement] Add 3d building color editor

### PR DESCRIPTION
**Pull request by heshan0131**
Screen Shot 2019-07-25 at 2 06 06 PM

Signed-off-by: Shan He <heshan0131@gmail.com>

### CODE REVIEW

**Comment by heshan0131** [comment]
https://github.com/keplergl/kepler.gl/issues/630

File: src/reducers/map-style-updaters.js [hunk 1]
```diff
@@ -179,28 +179,36 @@ export function getMapStyles({
...
+
   // brighten or darken building based on style
   const operation = style.id.match(/(?=(dark|night))/) ? 'brighter' : 'darker';
+
   const alpha = 0.2;
   const rgbObj = rgb(buildingColor)[operation]([alpha]);
   return [rgbObj.r, rgbObj.g, rgbObj.b];
 }
 
 function getLayerGroupsFromStyle(style) {
```
Comment (heshan0131) [inline src/reducers/map-style-updaters.js]: I ran prettier on this file

File: src/reducers/map-style-updaters.js [hunk 2]
```diff
@@ -179,28 +179,36 @@ export function getMapStyles({
   return {bottomMapStyle, topMapStyle, editable, threeDBuildingColor};
 }
 
+function findLayerFillColor(layer) {
+  return layer && layer.paint && layer.paint['background-color'];
+}
 function get3DBuildingColor(style) {
```
Comment (macrigiuseppe) [inline src/reducers/map-style-updaters.js]: on empty line here

**Comment by eportico** [comment]
Amazing! 
thanks a lot

**Pull request merged**